transport/wseb/src/test/scripts/org/kaazing/gateway/transport/wseb/send404OnInvalidDownstreamToProtectedService.rpt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2007-2015, Kaazing Corporation. All rights reserved.
+#
+
+connect http://localhost:8001/echoAuth/;e/db
```

Comment:
Do we need extra test case where they put in a sessionid which is no longer valid (e.g. /echo/;e/utm/waGcY3k5CDLMxy0mNvijdpXRtHxFjt5Z or /echo/;e/utm/waGcY3k5CDLMxy0mNvijdpXRtHxFjt5Z&.krn=0.7393842079985082)?